hivemq-edge/src/main/java/com/hivemq/security/ssl/SslUtil.java

```diff
@@ -162,6 +173,97 @@ private static byte[] loadFileContentAndConvertIfBase64Encoded(@NotNull String k
         }
     }
 
+    public static @NotNull ImmutableSet<X509Certificate> loadCertificatesFromTruststore(final @Nullable String password,
+                                                                                       final @Nullable String type,
+                                                                                       final @Nullable Path path)
```

Comment:
the bottom 3 methods of class `SslUtil` are not being used:
- loadCertificatesFromTruststore
- loadSSLContextFromTruststore
- loadSSLContextFromTruststore
they can be removed safely.